### PR TITLE
[39] added actions config to mfinit

### LINKED ISSUE #39: metaflowtask-init to create actions config as well

### PR DESCRIPTION
**Pull request by jaklinger**
Closes #39 

Makes sure that when `metaflowtask-init` is run that corresponding new DAPS repo has information for finding a test version of the config for actions

### CODE REVIEW

File: daps_utils/__initplus__.py [hunk 1]
```diff
@@ -15,14 +17,14 @@ def path_to_this(this):
...
-    """Load a config file from the given path."""    
+    """Load a config file from the given path."""
     import yaml, json, configparser
     with open(path) as f:
         if path.suffix in ('.yml', '.yaml'):
             return yaml.safe_load(f)
         if path.suffix == '.json':
             return json.load(f)
-        if path.suffix in ('.cnf', '.conf', '.config'):
+        if path.suffix in ('.cnf', '.conf', '.config', '.cfg'):
```
Comment (jaklinger) [inline daps_utils/__initplus__.py:27]: also adding this for luigi config

File: daps_utils/__initplus__.py [hunk 2]
```diff
@@ -47,6 +49,9 @@ def recursive_load(path_to_config):
 
 def load_config():
     """Load all of the config files"""
+    config_name = 'config'
+    if 'GITHUB_ACTIONS' in os.environ:  # CI/CD flag, set by default in GH Actions
```
Comment (jaklinger) [inline daps_utils/__initplus__.py:53]: implies that in "GH actions" mode

File: daps_utils/__initplus__.py [hunk 3]
```diff
@@ -47,6 +49,9 @@ def recursive_load(path_to_config):
...
+        config_name += '/actions-config'
```
Comment (jaklinger) [inline daps_utils/__initplus__.py:54]: find the config under this directory if in "GH actions" mode

File: scripts/metaflowtask-init [hunk 4]
```diff
@@ -74,7 +74,8 @@ if [ "${PWD##*/}" != "${DU_LOCAL_REPO_NAME}" ]; then
 fi
 # Set up MetaflowTask structure
 cd ${DU_LOCAL_REPO_NAME}
-CreateConfig ${DU_LOCAL_REPO_NAME}
+CreateConfig ${DU_LOCAL_REPO_NAME} config
+CreateConfig ${DU_LOCAL_REPO_NAME} config/actions-config
```
Comment (jaklinger) [inline scripts/metaflowtask-init:78]: Also add `config/actions-config`

**Review by bishax — COMMENTED** [review]
Changes are good but relies on changes in #50 being merged in order to work.

**Pull request merged**